mage.go

```diff
@@ -0,0 +1,49 @@
+//go:build mage
+
+package main
+
+import (
+	"fmt"
+	"github.com/magefile/mage/sh"
+)
+
+const versionStringEnvVarName = "POCKET_VERSION_STRING"
+const pocketPackage = "github.com/pokt-network/pocket/cmd/pocket"
+const ldflag = "-X main.version=$" + versionStringEnvVarName
+
+var env = map[string]string{}
+
+func init() {
+	env["GO111MODULE"] = "on"
+
+	fmt.Println("Getting branch and commit information")
+	branch, _ := sh.Output("git", "branch", "--show-current")
+	hash, _ := sh.Output("git", "rev-parse", "--short", "HEAD")
+	_, dirty := sh.Output("git", "diff", "--quiet")
+
+	if branch != "" {
+		branch = "-" + branch
+		hash = "/" + hash
+	}
+	if dirty != nil && dirty.Error() == "running \"git diff --quiet\" failed with exit code 1" {
```

Comment:
Maybe moving this long string into an unexported constant?

Reply:
it's used in a single place; I generally prefer to extract into declarations when the information is shared across places, otherwise it's just extra steps.